src/v/ssx/abort_source.h

```diff
@@ -38,6 +38,8 @@ class sharded_abort_source {
         });
     }
 
+    ss::future<> start() { return _as.start(); }
```

Comment:
Please document this public method.

Reply:
This commit will likely be deleted soon.

Reply:
SG, let's delete it.